### PR TITLE
doc: prep for 0.3 release

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

**Review by ljharb — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "@pkgjs/support",
-  "version": "0.0.2",
+  "version": "0.0.3",
   "description": "Package support information (see: https://github.com/nodejs/package-maintenance/pull/220)",
-  "author": "Wes Todd <wes@wesleytodd.com>",
+  "author": "pkgjs contributors - https://github.com/pkgjs/support/graphs/contributors",
```
Comment (wesleytodd) [inline package.json]: does this work?  I think we will want to use the contributors field and still have a normal author.  I am pretty sure npm uses this to link to publishers account, right?

Comment (ljharb) [reply package.json]: oh right, this doesn't work. "author" has to be the same as the contents of "contributors" - a name, with an optional email and optional URL.

Comment (ljharb) [reply package.json]: that said, no, there's no link here to any npm accounts, this is just not valid according to the schema for package.json.

Comment (mhdawson) [reply package.json]: probably not, should I leave the author as is for now ?

Comment (mhdawson) [inline package.json]:
```suggestion
"author": "Wes Todd <wes@wesleytodd.com>",
```

File: package.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "@pkgjs/support",
-  "version": "0.0.2",
+  "version": "0.0.3",
   "description": "Package support information (see: https://github.com/nodejs/package-maintenance/pull/220)",
-  "author": "Wes Todd <wes@wesleytodd.com>",
+"author": "Wes Todd <wes@wesleytodd.com>",
```
Comment (mhdawson) [inline package.json]:
```suggestion
  "author": "Wes Todd <wes@wesleytodd.com>",
```

File: package.json (hunk 1 above)
Comment (mhdawson) [reply package.json]: ok just removed that, want to be ready to get the blog post out and need a publish for that, we can figure out the author field later on.

Comment (wesleytodd) [reply package.json]: Yeah, that makes sense at first.  I think we should discuss the general strategy here for the org, but it seems to me we just add each new contributor ad-hoc?

**Review by wesleytodd — APPROVED** [review]
Not that it matters much, but I would like to get us on some shared tooling for reving versions and publishing at some point.  Not a deal now, but something to consider for the future.

**Pull request merged**